integration/query_compat_test.go

```diff
@@ -229,6 +229,14 @@ func TestQueryCompatSort(t *testing.T) {
 			filter: bson.D{},
 			sort:   bson.D{{"v", -1}, {"_id", 1}},
 		},
+		"AscSingle": {
+			filter: bson.D{},
+			sort:   bson.D{{"v", 1}},
+		},
+		"DescSingle": {
+			filter: bson.D{},
+			sort:   bson.D{{"v", -1}},
+		},
```

Comment:
CI is not happy https://github.com/FerretDB/FerretDB/actions/runs/4927425799/jobs/8807704213?pr=2595

`v` alone without `_id` won't work because in our shareddata, we have multiple documents that contains the same sort order of `v`. For example `int64(42)`, `42.0` and `int32(42)` are considered the same sorting order in bson. That's why we have `Asc` and `Desc` test cases with `v` and `_id`, if `v` is the same it uses `_id`. :hugs:

Reply:
Nice catch! Fixed